doc/Public_API.md

```diff
@@ -16,7 +16,7 @@ Could be used to setup probes on kubernetes or docker-compose.
 #### docker-compose probe setup
 ```yaml
 healthcheck:
-    test: ["CMD-SHELL", "curl --fail http://localhost/platform/api/modules/health/live"]
+    test: ["CMD-SHELL", "curl --fail http://localhost:${PLATFORM_LISTENING_PORT:-8080}/platform/api/modules/health/live"]
```

Comment:
I think you can use `CDK_LISTENING_PORT` directly as latest version support it (with fallback on `PLATFORM_LISTENING_PORT` also)